tests/Mediator/constants.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0"?>
```

Comment:
Why add constants.xml in Mediator test folder?

Reply:
To override constants for some unit tests.
But anyway Mediator is disabled for now.
We will change this test some other time.